### DISCUSSION TITLE
Broken Link to Composer Install

### DISCUSSION THREAD

**Discussion by kd4e** [wintercms/winter#984] [General]
Clicking on Composer Install here: (https://github.com/wintercms/web-installer)

Takes you here:
(https://wintercms.com/docs/v1.2/docs/console/commands#console-install-composer)

Which displays this ...
![image](https://github.com/wintercms/winter/assets/74081053/a4312332-3152-42b3-bd50-df3ba01c1e67)

**Comment by bennothommo** [comment]
Thanks @kd4e - fixed on the README :)
